src/tinypeel/Peeling/PeelingIO.py

```diff
@@ -68,6 +68,7 @@ def writeOutParamaters(peelingInfo):
 
     np.savetxt(args.out + ".geno_error_prob.txt", peelingInfo.genoError, fmt="%f")
     np.savetxt(args.out + ".seq_error_prob.txt", peelingInfo.seqError, fmt="%f")
+    np.savetxt(args.out + ".rec_prob.txt", np.empty((1,1)), fmt="%f") # not be realized, just as a placeholder
```

Comment:
@AprilYUZhang Please open an issue so we don't forget to come back to this code. It's easy to forget such tiny TODOs in the code. Also, we are likely storing recombination length of the chromosome and use it to calculate recombination rate between loci - it is that rate that should be outputted here. Make a note of this in the issue.